src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="form">
+      <fieldset class="allFieldset">
```

Comment:
for css common convention to name classes in kebab-case or little another convention (for example BEM)
please, check all such naming